HISTORY.md

```diff
@@ -0,0 +1,21 @@
+Anserini
```

Comment:
`docs/release-notes/release-notes-0.1.0.md` ?

Reply:
You prefer each release have it's own separate release notes, as opposed to everything all gathered together in a file?

Reply:
I suppose if we wanted to go this route, we can create a new "Release History" section in the README.

Reply:
We could have a release history link in README.md 
And I am fine with release history in a single file. But let's put it under `docs/`?